pages/folder.tsx

```diff
@@ -0,0 +1,248 @@
+import { useState, useEffect, useRef } from 'react';
+import { getFoldersById, getLinksById } from '../3-weekly-mission/pages/api/api';

```

Comment:
<img width="571" alt="image" src="https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/assets/59413128/0aab2a77-12be-4ec6-bd60-9ab5a6e32549">

정확하지 않은 경로에서 import 하고 있어서 타입 에러가 나고 있네요. 
아래처럼 수정하시면 해결됩니다. 

```suggestion
import { getFoldersById, getLinksById } from './api/hello';
```